src/component/tooltip/TooltipHTMLContent.ts

```diff
@@ -60,17 +60,21 @@ function mirrorPos(pos: string): string {
 }
 
 function assembleArrow(
-    backgroundColor: ColorString,
+    tooltipModel: Model<TooltipOption>,
     borderColor: ZRColor,
     arrowPosition: TooltipOption['position']
 ) {
+    const backgroundColor = tooltipModel.get('backgroundColor');
+    const borderWidth = tooltipModel.get('borderWidth');
+
```

Comment:
Just notice a very small thing, could it move this after `if` statement? Since it only affect whatever after `if`.

Reply:
Fixed :exclamation: